containers/versions.yml

```diff
@@ -1,6 +1,8 @@
 debian_version:               "9.4"
 docker_version:               "17.05.0~ce-0~debian-stretch"
 geth_version:                 "v1.8.2"
+golem_version:                "0.13.0"
+concent_version:              "v0.6.0-39-gb882ecc"
```

Comment:
So it's possible to use the output of `git describe` with `git checkout`? Cool. I did not expect that.